client/src/components/User.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const User = ({ user }) => {
+  return (
+    <div className='user'>
+      <h2>{user.id}</h2>
+      <h2>{user.created_at}</h2>
+      <h2>{user.username}</h2>
+      <h2>{user.password}</h2>
```

Comment:
even though it's a hash, are you sure you want to be displaying this in the app?

Reply:
Yeah - I wouldn't display a password in a real app, haha.